04.android/06.solving-problems/15.multiple-profiles-issue/docs.en.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Problem with reinstalling AdGuard on Android devices when using the app on more than one user account'
```

Comment:
Better use "multiple users" term or "user profiles" instead of "account". It will sound less confusing this way.
Same for the Russian version.

Reply:
I agree that we should follow the official Android documentation. [Here](https://source.android.com/devices/tech/admin/multi-user) they state that a profile is a subset, 'A profile is a subset of, and tied to, the existence of a user. A user can have multiple profiles. Profiles are created through a Device Administration application. A profile always has an immutable association to a parent user, defined by the user that created the profile. Profiles do not exist beyond the lifetime of the creating user.'

If it is possible to install/uninstall AdGuard for multiple user profiles, we should state that. 

Please explain why we should not use "user accounts" for Android? What exactly makes this definition confusing? It may be useful to refer to the same link above.

Reply:
Maybe that's only my opinion, but it's not semantically clear what account is meant in this sentence (it can be any social account, such as Google). That's why I suggested to use that term.